### DISCUSSION TITLE
Form bookmarks for different form layouts for different purposes

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13423] [Feature Requests]
Just like you can have different index-view layouts that can be bookmarked, how about form layouts that are different for different purposes / users. Maybe this would hint towards permissions for the different forms.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
